### DISCUSSION TITLE
Admin Panel: Custom content in relationship tab

### DISCUSSION THREAD

**Discussion by Wulfheart** [filamentphp/filament#5948] [Help] [answered]
Hello,

I would like to display custom content in a tab of a resource in a tab where also the relationships are.
Is this possible or should I just create a dedicated page for this?

**Comment by danharrin** [comment, marked as answer]
I don't think its possible to add more tabs to the relation managers at the moment

**Reply by Wulfheart** [reply]
Thank you really much. Is it in the scope for v3?

**Reply by danharrin** [reply]
I don't know what the API would look like for it at all, so no
